is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
```

Comment:
I am not sure of the purpose of both slice and forEach on this line of code.
Please clarify.